### PR TITLE
Fix bazel test

### PR DESCRIPTION
**Pull request by krzyzacy**
#2270

### CODE REVIEW

**Comment by pipejakob** [comment]
The title is "fixing bazel test," but the diff contents look like "adding kubeadm e2e pull job back in," and I'm not sure how the two are related. Did the merge of #2270 not apply cleanly and drop off the `config.yaml` changes? Did another commit remove this job's config entry?

File: prow/config.yaml [hunk 1]
```diff
@@ -256,6 +256,48 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:
+    - name: pull-kubernetes-e2e-kubeadm-gce
```
Comment (ixdy) [inline prow/config.yaml]: this should probably be pull-security-kubernetes-... right?

**Comment by krzyzacy** [comment]
@pipejakob https://github.com/kubernetes/test-infra/pull/2255 is merged between last presubmit run for #2270 and it's being merged, which made a copy of all presubmit to k-s repo.

File: prow/config.yaml [hunk 2]
```diff
@@ -256,6 +256,48 @@ presubmits:
...
+            mountPath: /etc/service-account
+            readOnly: true
+          - name: ssh
+            mountPath: /etc/ssh-key-secret
+            readOnly: true
+          - name: cache-ssd
+            mountPath: /root/.cache
+        volumes:
+        - name: service
+          secret:
```
Comment (ixdy) [inline prow/config.yaml]: are these the right secrets for the kubernetes-security jobs?

Comment (krzyzacy) [inline prow/config.yaml]: They are using the same service account now https://github.com/kubernetes/test-infra/pull/2301

File: prow/config.yaml (hunk 1 above)
Comment (krzyzacy) [inline prow/config.yaml]: done

**Comment by krzyzacy** [comment]
I think it's using a different ssh key, so it would not actually success, however the kubeadm job is not a always-trigger, so it will be something to fix later. This PR is merely fixing the general presubmit in test-infra. @pipejakob @jessfraz

**Comment by krzyzacy** [comment]
and it's more broken :-(

**Comment by krzyzacy** [comment]
and it's proven copy-paste is a bad idea :imp:

**Comment by krzyzacy** [comment]
giving up, use the lazy way

File: prow/config/config_test.go [hunk 3]
```diff
@@ -49,6 +49,7 @@ func TestConfigSecurityJobsMatch(t *testing.T) {
 		j.Trigger = strings.Replace(j.Trigger, "pull-kubernetes", "pull-security-kubernetes", -1)
 		j.Context = strings.Replace(j.Context, "pull-kubernetes", "pull-security-kubernetes", -1)
 		j.re = sp[i].re
+		j.RunAfterSuccess = sp[i].RunAfterSuccess
```
Comment (ixdy) [inline prow/config/config_test.go]: er.

**Comment by krzyzacy** [comment]
Okay now this is less `er...`

**Comment by krzyzacy** [comment]
I'd like to get this fix in asap since currently all test-infra presubmit are sriracha red :red_circle:

/assign @fejta

**Pull request merged**